netlify.toml

```diff
@@ -4,7 +4,7 @@
     publish = "docs/book/book"
 
 [build.environment]
-    GO_VERSION = "1.21"
+    GO_VERSION = "1.22.2"
```

Comment:
If possible I would only pin the minor version

(but maybe it's not possible anymore, then it's fine)

Reply:
that's not possible, it would fail...